.github/workflows/test.yml

```diff
@@ -55,7 +55,14 @@ jobs:
           make setup-env
           make init
 
-      - name: Run tests
+      - name: Run pydeck tests
         run: |
           cd bindings/pydeck
           make test
+
+      - name: Run pydeck-carto tests
+        run: |
+          cd bindings/pydeck-carto
+          make init
+          make lint
```

Comment:
Introduce `make ci` which executes `lint` & `test`? Makes it consistent with the JS and clearer from the Makefile what CI will run